tests/lib/rules/newline-between-statements.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * @fileoverview Tests for newline-between-statements rule.
+ * @author Toru Nagashima
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/newline-between-statements");
+const RuleTester = require("../../../lib/testers/rule-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+
+ruleTester.run("newline-between-statements", rule, {
```

Comment:
For compatibility, can you include all of the tests from the JSCS rule with the associated configuration to make them pass?

That will help ensure this rule will support the edge cases that JSCS handled: https://github.com/jscs-dev/node-jscs/blob/master/test/specs/rules/require-padding-newlines-after-blocks.js#L50